index.js

```diff
@@ -97,7 +97,7 @@ class KeyringController extends EventEmitter {
    * either as a string or an array of UTF-8 bytes that represent the string.
    * @returns {Promise<Object>} A Promise that resolves to the state.
    */
-  createNewVaultAndRestore(password, seedPhrase) {
+  async createNewVaultAndRestore(password, seedPhrase) {
```

Comment:
Nit: In an `async` function it's more idiomatic to throw errors rather than return rejections, so we should probably update the two rejections below here as well

Reply:
Good shout -- I did this elsewhere but not here.  Fixed.